frontend/src/components/grantaccess/GrantAccess.vue

```diff
@@ -211,16 +209,16 @@ function areVerificationsPassed() {
 
 function toRequestPayload(
     formData: any,
-    forestClientNumber: FamForestClient | null
```

Comment:
it's confused to call this parameter forestClientNumber when it's not just the number, but also has the status and other information, renamed it